From 62b71d33e30a0e9b48c3d0e7657016a39882de14 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Tue, 27 Dec 2011 16:03:15 +0800
Subject: [PATCH 05/39] ENGR00170961 Add ASF, AVI, and WMV format into mtp
 device info

add freescale extension media file format support. Currently
mkv, flv media format is not define in MTP spec which cannot be
supported in MTP.

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 media/java/android/mtp/MtpDatabase.java | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/media/java/android/mtp/MtpDatabase.java b/media/java/android/mtp/MtpDatabase.java
index cf1238a..b16ce20 100755
--- a/media/java/android/mtp/MtpDatabase.java
+++ b/media/java/android/mtp/MtpDatabase.java
@@ -512,6 +512,10 @@ public class MtpDatabase {
             MtpConstants.FORMAT_PLS_PLAYLIST,
             MtpConstants.FORMAT_XML_DOCUMENT,
             MtpConstants.FORMAT_FLAC,
+            // add fsl supported medial format
+            MtpConstants.FORMAT_ASF,
+            MtpConstants.FORMAT_AVI,
+            MtpConstants.FORMAT_WMV,
         };
     }
 
-- 
1.8.0

